.travis.yml

```diff
@@ -1,10 +1,61 @@
-language: shell
+language: rust
```

Comment:
I was avoiding this because we don't have control over the version that gets installed, and even if we did it would be different from the one we have in Docker. I'd rather have all the bits in one place, and I was hoping that place could be the docker image, mainly because it's easy to run that locally -- AFAICT it's not possible to "run" travis locally

Reply:
I've dropped the run-tests commit, particularly given that #105 is more Docker-based.